[2025-08-16 21:25:33,867] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.create_site_codes manual__2025-08-16T21:25:27.221750+00:00 [queued]>
[2025-08-16 21:25:33,878] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.create_site_codes manual__2025-08-16T21:25:27.221750+00:00 [queued]>
[2025-08-16 21:25:33,879] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 21:25:33,880] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-16 21:25:33,880] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 21:25:33,890] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_site_codes> on 2025-08-16 21:25:27.221750+00:00
[2025-08-16 21:25:33,898] {standard_task_runner.py:52} INFO - Started process 339 to run task
[2025-08-16 21:25:33,901] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '1_load_initial_data', 'create_site_codes', 'manual__2025-08-16T21:25:27.221750+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/initialise_data.py', '--cfg-path', '/tmp/tmpzo16ed4i', '--error-file', '/tmp/tmphx308u3q']
[2025-08-16 21:25:33,903] {standard_task_runner.py:80} INFO - Job 14: Subtask create_site_codes
[2025-08-16 21:25:33,967] {task_command.py:369} INFO - Running <TaskInstance: 1_load_initial_data.create_site_codes manual__2025-08-16T21:25:27.221750+00:00 [running]> on host 6a6e6aafe640
[2025-08-16 21:25:34,057] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=1_load_initial_data
AIRFLOW_CTX_TASK_ID=create_site_codes
AIRFLOW_CTX_EXECUTION_DATE=2025-08-16T21:25:27.221750+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-16T21:25:27.221750+00:00
[2025-08-16 21:25:34,063] {sql.py:262} INFO - Executing: 
    CREATE TABLE IF NOT EXISTS dbt_raw_data.site_codes (
        site_code CHAR(3) PRIMARY KEY,
        city_name VARCHAR(128),
        country_name VARCHAR(128) NULL,
        continent CHAR(2),
        iso_country CHAR(2) NULL
    );
    
[2025-08-16 21:25:34,097] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 21:25:34,160] {logging_mixin.py:115} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/connection.py:339 DeprecationWarning: The "schema" arg has been renamed to "database" as it contained the database name.Please use "database" to set the database name.
[2025-08-16 21:25:34,174] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 21:25:34,199] {sql.py:375} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS dbt_raw_data.site_codes (
        site_code CHAR(3) PRIMARY KEY,
        city_name VARCHAR(128),
        country_name VARCHAR(128) NULL,
        continent CHAR(2),
        iso_country CHAR(2) NULL
    );
    , parameters: None
[2025-08-16 21:25:34,232] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=1_load_initial_data, task_id=create_site_codes, execution_date=20250816T212527, start_date=20250816T212533, end_date=20250816T212534
[2025-08-16 21:25:34,277] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-16 21:25:34,328] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
